src/EnsembleKalmanProcess.jl

```diff
@@ -127,6 +131,7 @@ function EnsembleKalmanProcess(
     rng::AbstractRNG = Random.GLOBAL_RNG,
     failure_handler_method::FM = IgnoreFailures(),
     localization_method::LM = NoLocalization(),
+    verbose::Bool = true,
```

Comment:
Is it common to have this as `false` by default? (as in theory people will only need it when checking convergence / debugging?